registry/remote/example_test.go

```diff
@@ -181,6 +185,56 @@ func ExampleRepository_Push() {
 	// Push finished
 }
 
+// ExampleRepository_Push_referenceManifest gives an example snippet for pushing a reference manifest.
+func ExampleRepository_Push_referenceManifest() {
+	reg, err := remote.NewRegistry(host)
+	if err != nil {
+		panic(err)
+	}
+	ctx := context.Background()
+	repo, err := reg.Repository(ctx, exampleRepositoryName)
+	if err != nil {
+		panic(err)
+	}
+
+	// 1. assemble the referenced manifest
+	rfedManifest := artifactspec.Manifest{
+		MediaType:    artifactspec.MediaTypeArtifactManifest,
+		ArtifactType: "referenced manifest",
+	}
+	rfedManifestContent, _ := json.Marshal(rfedManifest)
+	rfedManifestDescriptor := artifactspec.Descriptor{
+		MediaType:    ocispec.MediaTypeImageManifest,
```

Comment:
The descriptor media type(ocispec.MediaTypeImageManifest) and manifest media type(artifactspec.MediaTypeArtifactManifest) should be the same.

Reply:
changed it.